### ISSUE TITLE
Add LCIA scores

### ISSUE THREAD

**Issue by cmutel** [cauldron/bw_matchbox#150]
We can get LCIA scores at the endpoint `scores/get/<id>`, e.g. `https://dare.cauldron.ch/scores/get/3456`. The result format is:

```json
[
  {
    "category": "string",
    "original": "float",
    "relinked": "float",
    "ratio": "float",
    "unit": "string"
  }
]
```

We need to make these scores available on the `process/<id>` page.

As this layout is already fixed, and scores don't need to be loaded eagerly, perhaps we could make this an additional window which expands in similar to the comments?

We would need another icon, something appropriate for scores or grades or test results.

In the list that is shown, the numbers can have a lot of digits, and can be scientific (e.g. `3.4e7`). We need a formatter that reduces the numbers of 3 digits (total, including before the period), while preserving the exponent if available.

The display order can be alphabetical by `category`.

Up to @lilliputten whether this is a list or a table.

**Comment by lilliputten** [comment]
@cmutel

I have questions only about the UI:

Let's use a common (the same) side panel, and use one of the next options to switch between comments and scores:

1. Use a common toggle button (with a generic icon) and inner tabs for 'Comments' and 'Scores' on the panel's top.
2. Use different icons outside, and add states to them (to accent the currently active one).

The first option seems a bit clearer (in the UI/UX sense) and easier to implement (if I'm not missing anything).

**Comment by cmutel** [comment]
To me the comments are very different, but of course they are related, as you use the scores for reviewing.

What about having one panel, with the scores populated when available in the top half and the comments in the bottom half. They can both overflow so need to be scrollable. But this is just an idea, the user story is that it would be more convenient to be able to see the scores and comments threads at the same time, I don't know how it's best to implement that.

**Comment by lilliputten** [comment]
I meant something like this for tabs:

![side-panel-tabs-v1-1](https://github.com/cauldron/bw_matchbox/assets/6855837/b0350eb6-f0c0-4da1-81c9-8ddb2d570c3d)

Do you think this will confuse the user, causing him to confuse comments and points?

**Comment by cmutel** [comment]
OK, let's go with your mock up. This is fine for now.

**Comment by lilliputten** [comment]
Now I see that it actually makes scores and comments a little closer, like similar things.

Is this ok? Are we still deciding to go this way?

**Comment by cmutel** [comment]
> Apparently, you forgot to tell about the relationship between the main process table and the list of ratings?
> Should there be any interactions between these data?

These are amounts of total environmental damage, and are not related to the main process table. No interaction.

> Should these tables be shown at the same time when we open the sidebar? (Is it ok -- to load data that probably will not be used?)

OK to load data even if it is not seen, it is cached.

> Or do we have to have two different control buttons in the main pane (to open them independently)?

I think one button is fine for now.

**Comment by cmutel** [comment]
> Is this ok? Are we still deciding to go this way?

Yes, this is OK. We can come back later if needed.

**Comment by lilliputten** [comment]
Has implemented in PR #151.

![20231205-001918](https://github.com/cauldron/bw_matchbox/assets/6855837/905060d4-8c8e-42f5-aba9-7370d0588b67)
